From eed757cee5a2814768b369e54e2ae96ebfca33f7 Mon Sep 17 00:00:00 2001
From: Steve Dickson <SteveD@redhat.com>
Date: Thu, 18 Sep 2014 14:19:31 -0400
Subject: [fs] nfs: Fix unused variable error

Message-id: <1411050057-12417-9-git-send-email-steved@redhat.com>
Patchwork-id: 96180
O-Subject: [RHEL7.1 PATCH 08/94] nfs: Fix unused variable error
Bugzilla: 1111708
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Benjamin Coddington <bcodding@redhat.com>

From: Anna Schumaker <Anna.Schumaker@netapp.com>

inode is unused when CONFIG_SUNRPC_DEBUG=n.

Signed-off-by: Anna Schumaker <Anna.Schumaker@Netapp.com>
Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
Upstream Id: 343ae531f12b046ddfe54b0aa91b11b257f0c55f
Bugzilla: 1111708
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfs/pagelist.c b/fs/nfs/pagelist.c
index 9097ee9..05a2402 100644
--- a/fs/nfs/pagelist.c
+++ b/fs/nfs/pagelist.c
@@ -552,7 +552,6 @@ static void nfs_pgio_prepare(struct rpc_task *task, void *calldata)
 int nfs_initiate_pgio(struct rpc_clnt *clnt, struct nfs_pgio_header *hdr,
         const struct rpc_call_ops *call_ops, int how, int flags)
 {
- struct inode *inode = hdr->inode;
  struct rpc_task *task;
  struct rpc_message msg = {
   .rpc_argp = &hdr->args,
@@ -575,8 +574,8 @@ int nfs_initiate_pgio(struct rpc_clnt *clnt, struct nfs_pgio_header *hdr,
  dprintk("NFS: %5u initiated pgio call "
   "(req %s/%llu, %u bytes @ offset %llu)\n",
   hdr->task.tk_pid,
-  inode->i_sb->s_id,
-  (unsigned long long)NFS_FILEID(inode),
+  hdr->inode->i_sb->s_id,
+  (unsigned long long)NFS_FILEID(hdr->inode),
   hdr->args.count,
   (unsigned long long)hdr->args.offset);
 
-- 
1.7.1